Core/MIPS/IR/IRCompVFPU.cpp

```diff
@@ -788,6 +788,8 @@ namespace MIPSComp {
 		case VecDo3Op::VSLT:
 			allowSIMD = false;
 			break;
+        default:
+            break;
```

Comment:
Ah, sorry.  Some indent things here and below.

I'd rather this was INVALID, though, but I guess we'll never add new ones here anyway.

-[Unknown]